docs/DeveloperGuide.md

```diff
@@ -1,34 +1,1135 @@
+    1. The execute() method will call print out the list of commands onto the console with printHelpFormatter() from static CommonUi.java.
+[&#8593; Return to Top](#developer-guide)
+
+#### 4.3.2. Start recording diet data: `new`
+The feature allows users to start recording diet data. 
+
+**Implementation**  
+When the user types `new </d [DATE]> </t [TAG]>` the following sequence occurs. 
+1. The user keys in `new /d 2020-05-04 /t breakfast`.
+    
+    1. A `DietSessionUi` component will call `dietSessionUI.getInput()`. 
+    1. Input will be parsed in `processCommand()`, which splits the input into the `command` and the `input`.   
+    
+2. Creation of command object from input
+    1. This will create an instantiation DietSessionCreate() command object of which the method execute() is called.
+    
+3. Executing Command
+    1. The newly created object will then create an instantiation of a `DietSession`, and call the `start()` method.
+
+The sequence diagram below summarizes how creating new diet session works:
+
+![Load Data Sequence Diagram](pictures/Shukai/CreateDietSession.png)
+
+[&#8593; Return to Top](#developer-guide)
+
+#### 4.3.2.1. Showing help message: `help`
+This command lists out all help commands in a typed list that indicates to the user all the commands available and how to use them.
+
+**Implementation**  
+When the user types `help` the following sequence occurs. 
+1. The user keys in `help`.
+    
+    1. A `DietSessionUi` instantiation calls `dietSessionUI.getInput()`. 
+    1. Input will be parsed in `processCommand()`.   
+    
+2. Creation of command object from input
+    1. This will create a `FoodItemHelp()` instantiation of which the method `execute()` is called.
+    
+3. Executing Command
+    1. The newly created object will then print out the list of commands onto the console with printHelpFormatter() from static CommonUi.java.
+    
+[&#8593; Return to Top](#developer-guide)
+
+#### 4.3.2.2. Adding food items for the current diet: `add`
+
+The feature allows users to add food items into the current diet session. 
+
+**Implementation**  
+When the user types `add [FOOD_NAME] /c [CALORIES]` the following sequence occurs. 
+1. The user keys in `add bologna /c 123`.
+    
+    1. A `DietSessionUi` component will call `dietSessionUI.getInput()`. 
+    1. Input will be parsed in `processCommand()`.   
+    
+2. Creation of command object from input
+    1. This will create a `FoodItemAdd()` instantiation of which the method `execute()` is called.
+    1. The food component `bologna` and calories component `123` are passed into the constructor of a Food instantiation.
+    
+3. Executing Command
+    1. The newly created food object will then be added to the `ArrayList<Food> foodList` in diet session.
+
+The sequence diagram below summarizes how adding a new food to the diet session works:
+
+![Load Data Sequence Diagram](pictures/Zeon/FoodItemAdd.png)
+[&#8593; Return to Top](#developer-guide)
```

Comment:
I think there is an indentation here? Since all of the 'Return to Top' is on a new line.